tests/store/_unity_catalog/model_registry/test_unity_catalog_rest_store.py

```diff
@@ -695,7 +697,10 @@ def test_create_model_version_gcp(store, local_model_dir, create_args):
         if "run_id" in create_kwargs:
             _, run = store._get_run_and_headers("some_run_id")
             notebook_id = store._get_notebook_id(run)
-            create_kwargs["extra_headers"] = {_DATABRICKS_LINEAGE_ID_HEADER: str(notebook_id)}
+            entity_info = EntityInfo(entity_type=NOTEBOOK, entity_id=str(notebook_id))
+            create_kwargs["extra_headers"] = {
+                _DATABRICKS_LINEAGE_ID_HEADER: entity_info.SerializeToString()
```

Comment:
Maybe test that this equals the expected JSON? 
```suggestion
                _DATABRICKS_LINEAGE_ID_HEADER: json.dumps({"entity_type": NOTEBOOK, "entity_id": str(notebook_id)})
```